### DISCUSSION TITLE
Error while building Nuxt3 stable

### DISCUSSION THREAD

**Discussion by duderion** [nuxt/nuxt#16359] [Questions]
firstofall: i appreciate your hard work on nuxt3. its an awesome project. and i also liked the nuxt nation yesterday.

now i wanted to upgrade from rc12 to final. butwhen i run yarn build i get the following

![image](https://user-images.githubusercontent.com/949964/202416045-62c9040b-6fde-4af9-8d97-acbc5d9d1166.png)

my package.json looks like this

{
  "name": "@pinia/nuxt",
  "version": "0.4.4",
  "description": "Nuxt Module for pinia",
  "keywords": [
    "pinia",
    "nuxt",
    "vue",
    "vuex",
    "store"
  ],
  "homepage": "https://github.com/vuejs/pinia/tree/v2/packages/nuxt#readme",
  "bugs": {
    "url": "https://github.com/vuejs/pinia/issues"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/vuejs/pinia.git"
  },
  "funding": "https://github.com/sponsors/posva",
  "license": "MIT",
  "author": {
    "name": "Eduardo San Martin Morote",
    "email": "posva13@gmail.com"
  },
  "sideEffects": false,
  "type": "module",
  "exports": {
    ".": {
      "import": "./dist/module.mjs",
      "require": "./dist/module.cjs"
    }
  },
  "main": "./dist/module.cjs",
  "types": "./dist/types.d.ts",
  "files": [
    "dist"
  ],
  "dependencies": {
    "@nuxt/kit": "^3.0.0",
    "pinia": ">=2.0.24"
  },
  "devDependencies": {
    "@nuxt/module-builder": "^0.2.1",
    "@nuxt/schema": "^3.0.0",
    "@nuxt/test-utils": "^3.0.0",
    "nuxt": "^3.0.0",
    "typescript": "^4.8.4",
    "vue-tsc": "^1.0.9"
  },
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "build": "pnpm dev:prepare && nuxt-module-build",
    "dev": "nuxi dev playground",
    "dev:build": "nuxi build playground",
    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
    "test:types": "pnpm dev:prepare && nuxi typecheck",
    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s --commit-path . -l @pinia/nuxt -r 1"
  }
}

Cheers 
Adrian

**Comment by IsraelOrtuno** [comment]
Seems like this method was removed after being deprecated in RC-14. Consider opening an issue or PR to that module and updating it by `addImports`

**Comment by duderion** [comment]
https://github.com/nuxt/framework/issues/9121
